Src/Couchbase.Linq.IntegrationTests/BucketContextTests.cs

```diff
@@ -367,6 +367,26 @@ public void SubmitChanges_WhenDocsAreModifiedAndEndChangeTracking_DocumentIsNotM
             var doc = ClusterHelper.GetBucket("beer-sample").GetDocument<Beer>(((ITrackedDocumentNode)beer).Metadata.Id);
             Assert.AreNotEqual(beer.Abv, doc.Content.Abv);
         }
+
+        [Test]
+        public void Underlying_Bucket_can_be_used_to_execute_N1QL_query_directly()
+        {
+            // setup bucket context
+            var db = new BucketContext(ClusterHelper.GetBucket("beer-sample"));
+
+            // get a beer from the db
+            var beer = db.Query<Beer>().Select(b => N1QlFunctions.Meta(b)).First();
+
+            // execute N1QL to update the beer
+            var newBeerName = Guid.NewGuid().ToString();
+            var n1ql = $"UPDATE `beer-sample` USE KEYS '{beer.Id}' SET name = '{newBeerName}';";
```

Comment:
@mgroves 

We're not yet supporting C# 6.0 language features in this project yet, it should still be able to build in VS 2013.

Brant